host/debian/files

```diff
@@ -0,0 +1,2 @@
+hss-dkms_0.1.0-1_all.deb admin optional
```

Comment:
What is the file (named "file"!) for?

Reply:
Nothing, it would seem (just a remnant from an example I copied). I removed it without consequence so I'll push that remove in.